src/timing.jl

```diff
@@ -9,9 +9,8 @@ export every, fpswhen, fps, timestamp
 function every(delta::Float64)
     i = Input(time())
     update(timer) = push!(i, time())
-    t = Timer(update)
-    start_timer(t, delta, delta)
-    return lift(x->x, i, typ=Float64) # prevent push!
+    t = Timer(update, delta, delta)
```

Comment:
https://github.com/JuliaLang/Compat.jl/pull/103 should handle this once it's merged.